src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -146,8 +148,15 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
-    deinitialize(): Promise<InternalCheckoutSelectors> {
+    async deinitialize(_options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
```

Comment:
if you are using the `options` parameter you should remove the underscore prefix